easybuild/easyconfigs/l/libMemcached/libMemcached-1.0.18-GCC-6.4.0-2.28.eb

```diff
@@ -0,0 +1,20 @@
+easyblock = 'ConfigureMake'
+
+name = 'libMemcached'
+version = '1.0.18'
+
+homepage = 'https://memcached.org'
+description = """libMemcached is an open source C/C++ client library and tools for the memcached server (http://danga.com/memcached). It has been designed to be light on memory usage, thread safe, and provide full access to server side methods."""
+
+toolchain = {'name': 'GCC', 'version': '6.4.0-2.28'}
```

Comment:
Any particular reason for favouring `GCC` over `GCCcore` toolchain?

Using the latter would make this compatible with both `foss` and `intel` toolchains (or anything else that uses `GCC(core)` as a base).

If you change to `GCCcore`, you'll need to include a `binutils` build dependency (preferably under the `sources` block):

```python
toolchain = {'name': 'GCCcore', 'version': '6.4.0'}

...

builddependencies = [('binutils', '2.28')]
```